content/docs/setup/kubernetes/quick-start-alicloud-ack/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Quick Start with AliCloud Kubernetes Container Service
+description: Quick Start instructions to setup the Istio service using AliCloud Kubernetes Container Service
+weight: 12
+keywords: [kubernetes,alicloud]
+---
+
+Quick Start instructions to install and run Istio in [AliCloud Kubernetes Container Service](https://cs.console.aliyun.com/) using Application Catalog.
+
+This Quick Start installs the current release version (0.8.0) of Istio and then deploys the [Bookinfo](/docs/guides/bookinfo/) sample
+application.  It uses Application Catalog to automate the steps detailed in the [Istio on Kubernetes setup guide](/docs/setup/kubernetes/quick-start/) for Kubernetes Engine.
+
+## Prerequisites
+
+- This sample assumes that you already have one avaiable AliCloud Kubernetes cluster. Otherwise, you can create a Kubernetes cluster quickly and easily in the Container Service console.
+
+
+- Make sure that `kubectl` can work fine for your Kubernetes cluster.
+
+- Create new namespace to deploy istio components, e.g. `istio-system`:
+
+```console
```

Comment:
This should be ```command, and the block should be indented by four spaces.

This applies to other command-lines.

Reply:
Thanks a lot. It should looks like this, right?
```command
    kubectl create namespace istio-system
```

Reply:
or this?
```command
$ kubectl create namespace istio-system
```